examples/polypharmacy_gripnet/utils.py

```diff
@@ -1,93 +1,84 @@
-import pickle
+import imp
+import os
 
 import numpy as np
 import torch
 from sklearn import metrics
-from sklearn.metrics import accuracy_score
+from torch.utils.data import DataLoader, Dataset
+from torch_geometric.data.data import Data
+from yacs.config import CfgNode
+
+from kale.prepdata.supergraph_construct import SuperVertexParaSetting
+from kale.utils.download import download_file_by_url
 
 EPS = 1e-13
```

Comment:
Consider: should this be configurable as a hyper parameter, or fixed?

Reply:
It is a small enough number, close to 0, used in loss function implementation to prevent problems in calculating gradients.

Reply:
I got what it is. My question is whether it should be configurable by user or not. Check some example references in PyTorch/PL to see

Reply:
It seems that it can be configurable. Therefore, I changed it to a parameter of `GripNetLinkPrediction()`.

Reply:
Can you add more docstrings to at least explain the arguments for `GripNetLinkPrediction` (see other code in PyKale)?